DiveLane/Presentation Layer/Controllers/WalletCreation/WalletCreationViewController.swift

```diff
@@ -300,6 +300,19 @@ class WalletCreationViewController: UIViewController {
         }
     }
     
+    func checkPasswordsMatch() -> Bool {
+        return !((!(passwordTextField.text?.isEmpty ?? true) ||
+            !(repeatPasswordTextField.text?.isEmpty ?? true)) &&
+            passwordTextField.text != repeatPasswordTextField.text) ||
+        repeatPasswordTextField.text?.isEmpty ?? true || passwordTextField.text?.isEmpty ?? true
+    }
+    
+    func passwordsMatch(_ match: Bool) {
```

Comment:
not obvious, what does this function do. The function should represent some verb.
 from the name. use prefix smth like `setColor`

Reply:
yes, sounds reasonable, I've fixed it in the last commit